src/pages/FrontendMentor/TipCalculatorApp/styles/TipCalculatorAppStyles.tsx

```diff
@@ -0,0 +1,248 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  background: hsl(185, 41%, 84%);
+  height: 100vh;
+`
+
+export const Header = styled.div`
+  padding-top: 10vh;
+  padding-bottom: 8vh;
+  text-align: center;
+`
+
+export const HeaderWords = styled.div`
+  color: hsl(183, 100%, 15%);
+  font-family: 'Space Mono';
+  font-weight: bold;
+  font-size: 18px;
+  margin-top: 2vh;
+  margin-bottom: 2vh;
+`
+
+export const MiddleContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  height: 50vh;
+  width: 100vw;
+  @media (max-width: 768px) {
+    height: 90vh;
+  }
+`
+
+export const Calculator = styled.div`
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  background: hsl(0, 0%, 100%);
+  width: 900px;
+  border-radius: 25px;
+  height: 90%;
+  @media (max-width: 768px) {
+    flex-direction: column;
+    align-items: center;
+  }
+`
+
+export const InteractionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  font-family: 'Space Mono';
+  width: 40%;
+  @media (max-width: 768px) {
+    width: 80%;
+  }
+`
+
+export const ItemContainer = styled.div`
+  padding-top: 0.5vh;
+`
+
+export const ItemHeaderContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+export const ItemHeader = styled.div`
+  font-weight: bold;
+  color: hsl(186, 14%, 43%);
+  padding-bottom: 1vh;
+`
+
+export const Warning = styled.div`
+  color: red;
+  font-family: 'Space Mono';
+  font-weight: bold;
+`
+
+export const InputContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 5vh;
+  line-height: 5vh;
+  background: hsl(189, 41%, 97%);
+  border-radius: 3px;
+  vertical-align: center;
+  &:hover {
+    border: hsl(172, 67%, 45%) solid 1px;
+  }
+`
+
+export const InputPeopleContainer = styled.div<{ peopleAmount: number }>`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 5vh;
+  line-height: 5vh;
+  background: hsl(189, 41%, 97%);
+  border-radius: 3px;
+  vertical-align: center;
+  ${(props) => props.peopleAmount <= 0 && 'border: solid red 1px;'}
+  &:hover {
+    border: ${(props) => props.peopleAmount <= 0 && 'hsl(172, 67%, 45%) solid 1px'};
```

Comment:
might want to include 'border' in the portion where props.peopleAmount <= 0 is true instead

Reply:
Resolved in 70e675c